src/containers/AboutIndex/components/MembersCard.tsx

```diff
@@ -0,0 +1,44 @@
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import makeStyles from '@material-ui/core/styles/makeStyles';
```

Comment:
Trodde ikke Material UI støttet importering fra nivå 3? https://material-ui.com/guides/minimizing-bundle-size/#option-1
```suggestion
import { makeStyles } from '@material-ui/core/styles';
```